Resources/Prototypes/_SpaceCats/Catalog/uplink_catalog.yml

```
@@ -16,82 +16,4 @@
-  description: uplink-syndicate-necopai-desc
-  icon: { sprite: /Textures/_SpaceCats/Objects/Fun/pai-neco.rsi, state: syndicate-icon-pai-off }
-  productEntity: SyndicatePersonalNecoAI
-  cost:
-    Telecrystal: 1
-  categories:
-  - UplinkAllies
-  conditions:
-  - !type:ListingLimitedStockCondition
-    stock: 1
\ No newline at end of file
+          - SurplusBundle
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)